frontend/src/modules/editor/selectTheme/views/editorThemeItemView.js

```diff
@@ -24,6 +24,16 @@ define(function(require){
     },
 
     postRender: function() {
+      var previewUrl = '/api/theme/preview/' + this.model.attributes.name + '/' + this.model.attributes.version;
```

Comment:
Very minor but could we use `model.get()` to access attributes here please, for consistency.